Dockerfile

```diff
@@ -9,9 +9,12 @@ RUN go get github.com/m-lab/ndt5-client-go/cmd/ndt5-client
 
 # Murakami image
```

Comment:
Do you mind documenting why there are two Dockerfiles and the purpose of each perhaps by customising the line at the top of each? You told me about this when we were in a videochat and I think it's useful to do that in the spirit of leaving the camp better than how we found it. (I'd do this myself using a suggestion, but GitHub does not allow me to provide suggestions that are far from the context of the diff.)

Reply:
This is a good suggestion. Dockerfile is used for standalone installs, while Dockerfile.template is used for Balena managed installs.

Reply:
Done.

Reply:
Thanks!